lib/fatal_error/Kconfig

```diff
@@ -6,7 +6,7 @@
 
 menuconfig RESET_ON_FATAL_ERROR
 	bool "Reset on fatal error"
-	default y if !DEBUG && !KERNEL_DEBUG
+	default y if !DEBUG && !KERNEL_DEBUG && !BOARD_NATIVE_POSIX
```

Comment:
should be `default y if !DEBUG && !KERNEL_DEBUG && !ARCH_POSIX && !QEMU_TARGET`

Reply:
I fixed this already for the upmerge https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1750/commits/98fd10f934f0f80daf05a6a2179980c755a50c58

Reply:
Added missing qemu part in the upmerge.

Reply:
I can add this to #1791